### PR TITLE
Fix querying IPs from existing servers

### PR DESCRIPTION
**Pull request by Syphdias**
The output from `openstack server show` command is different when using
`-f value`. This means that the assumptions for the awk commands to
extract the IPs are off.

To fix this I used json formatting instead and `jq` to extract the IPs
more easily. In one case I decided to hard code the internal DNS IP
since it was set earlier to be something specific, so no need to query
it.

For the adjustments in /etc/hosts and keyscanning I added a filter for
the DOMAIN in case there are other servers in the project.

### CODE REVIEW

**Review by e-minguez — CHANGES_REQUESTED** [review]
Minor comment about avoid hardcoding the IP👍
Otherwise LGTM!

File: docs/03-compute-resources.md [hunk 1]
```diff
@@ -367,8 +368,7 @@ service:
 
 ```
 DOMAIN="k8s.lan"
-DNS_INTERNAL_IP=$(openstack server show dns.${DOMAIN} -f value -c addresses | awk -F'[=,]' '{print $2}')
-openstack subnet set --dns-nameserver ${DNS_INTERNAL_IP} kubernetes
+openstack subnet set --dns-nameserver 10.240.0.100 kubernetes
```
[the remark this answers is not in the document]
Comment (Syphdias) [reply docs/03-compute-resources.md]: Resolved

**Comment by Syphdias** [comment]
I added a section for CLI perquisites, mainly for `jq` but I also listed `ssh` and `scp`. Is there anything I should add?

**Comment by e-minguez** [comment]
That's awesome, thanks! 😃

**Review by e-minguez — APPROVED** [review]

**Pull request merged**